src/inference/src/os/cpu_map_info.hpp

```diff
@@ -24,16 +24,24 @@ class CPU {
     int _numa_nodes = 0;
     int _sockets = 0;
     int _cores = 0;
+    std::vector<std::vector<int>> _org_proc_type_table;
     std::vector<std::vector<int>> _proc_type_table;
     std::vector<std::vector<int>> _cpu_mapping_table;
     std::mutex _cpu_mutex;
-    int _plugin_status = PLUGIN_USED_START;
+    std::mutex _plugin_mutex;
     int _socket_idx = 0;
     int _num_threads = 0;
 };
 
 CPU& cpu_info();
 
+void reserve_cpu_by_streams_info(const std::vector<std::vector<int>> _streams_info_table,
```

Comment:
1, This function misses description
2. Why it is not declared in `system_conf.hpp`?

Reply:
Added function description. 
This function is to get _stream_processors by _streams_info_table and _cpu_mapping_table which is defined in cpu_map_info.hpp, and it is not a function intended to be provided to external calls. So it is not declared in system_conf.hpp.

Reply:
Then the question is why definition is placed inside `system_conf.cpp`?